.github/actions/release-build/shared-library-Linux.sh

```diff
@@ -2,15 +2,11 @@
 # This uses bash variable substitution in a few places
 # 1. getting the cmake directory for running cpack with an absolute path (chocolatey has an unfortunately named alias)
 
-#wget --quiet -O cmake-install.sh https://github.com/Kitware/CMake/releases/download/v3.15.5/cmake-3.15.5-Linux-x86_64.sh && chmod +x cmake-install.sh
-#sudo ./cmake-install.sh --prefix=/usr/local --exclude-subdir --skip-license
-#rm cmake-install.sh
-
 # Unset VCPKG_ROOT for GitHub actions environment
 unset VCPKG_ROOT
 
 mkdir build && cd build || exit
-cmake -DCMAKE_BUILD_TYPE=Release -DHELICS_ZMQ_SUBPROJECT=ON -DHELICS_ENABLE_PACKAGE_BUILD=ON -DSTATIC_STANDARD_LIB=static -DHELICS_BUILD_APP_EXECUTABLES=OFF -DHELICS_BUILD_APP_LIBRARY=OFF -DHELICS_BUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF ..
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -DHELICS_ZMQ_SUBPROJECT=ON -DHELICS_ENABLE_PACKAGE_BUILD=ON -DSTATIC_STANDARD_LIB=static -DHELICS_BUILD_APP_EXECUTABLES=OFF -DHELICS_BUILD_APP_LIBRARY=OFF -DHELICS_BUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF ..
```

Comment:
note that for the next release C++20 will be required, so might have to do some updates

Reply:
Yea, I think we'll need to bump up the minimum glibc required for the pre-compiled Linux binaries for the next release. Maybe making use of cibuildwheel where possible for managing execution of build commands across different OSes (and architectures).